### PR TITLE
:recycle: Fix flex box

### PR DESCRIPTION
**Pull request by zhumeisongsong**
Related #2
In this PR:
- Share if I make the layout, How I do it
image

### CODE REVIEW

**Comment by hiroki0404508** [comment]
Sorry for late!💦
I have some questions, and please talk about them:
- What is the relation of `header-container` and `wrapper`?
- In HTML and CSS, is it unusual to use camel case?
- You renamed `left-container` to `profile-container`. Is the reason why you did it for easy understanding to every person?
- At line 97, why `flex-direction` is `column`?

**Comment by zhumeisongsong** [comment]
@hiroki0404508 
>What is the relation of header-container and wrapper?

Personally, I like to use 2 div to control a container's layout, you can try to remove  `header-container` or` wrapper` and see the change.

>In HTML and CSS, is it unusual to use camel case?

Yes,  not use camel case when add class name usually

>You renamed left-container to profile-container. Is the reason why you did it for easy understanding to every person?

There is an **object-oriented** concept here
left or right doesn't describe what this container or object is，so not recommend. We will talk about this again in media queries, you will make sense then

File: hiroki/css-flex-box/stylesheet.css [hunk 1]
```diff
@@ -1,69 +1,144 @@
...
+  background: url("https://avatars.githubusercontent.com/u/103034701?v=4") #666
+    center no-repeat;
+  background-size: contain;
 }
 
-div.footerContainer {
+.content-container {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
```
Comment (zhumeisongsong) [inline hiroki/css-flex-box/stylesheet.css:97]: @hiroki0404508 
> At line 97, why flex-direction is column?
How about try to remove it and see what will happen?

image

Comment (zhumeisongsong) [reply hiroki/css-flex-box/stylesheet.css:97]: You can add single comment next to the line by this way
image

Comment (hiroki0404508) [reply hiroki/css-flex-box/stylesheet.css:97]: Thanks for comments!
But I am now worried about how to fetch your branch...
I tried `fit fetch origin feature/flex-box`, but got a message of `fatal: couldn't find remote ref feature/flex-box`...

Comment (zhumeisongsong) [reply hiroki/css-flex-box/stylesheet.css:97]: Let's have a short talk！

Comment (hiroki0404508) [reply hiroki/css-flex-box/stylesheet.css:97]: I understood why flex-direction is column at L97!

File: hiroki/css-flex-box/stylesheet.css [hunk 2]
```diff
@@ -1,69 +1,144 @@
...
+  list-style-type: none; /* merge to one line */
 }
 
-div.headerContainer {
+.wrapper {
   display: flex;
-  flex-direction: row;
   width: 100%;
-  justify-content: flex-start;
+  margin: 0 32px; /* Please confirm Why I add it */
```
Comment (hiroki0404508) [inline hiroki/css-flex-box/stylesheet.css:25]: To set a margin at left side of the icon?

**Review by hiroki0404508 — APPROVED** [review]
I mostly understood about flex-box, I think!
So, shall we go to the next step tomorrow?

**Pull request merged**